src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -29,6 +31,15 @@ class LocateControl extends PureComponent<{ leaflet: LeafletContext }> {
         lc.addTo(map as Leaflet.Map);
     }
 
+    componentDidMount() {
+        // This allows the page not to crash when the map is loaded
+        // But the locate button does not appear with error `Control.Locate is not a constructor`
```

Comment:
Can you try figuring out why the error is appearing so we don't have to do this setTimeout hack?

Reply:
Oh I was able to resolve this on my end (now pushed to main) by adding `import 'leaflet.locatecontrol'` after importing the other leaflet stuff